docs/web-configurator.md

```diff
@@ -286,6 +288,16 @@ Enabling this add-on will allow you to use GP2040-CE on a PS4 with an 8 minute t
 * `Serial Number (16 Bytes in Hex Ascii)` - Choose your serial number file.
 * `Signature (256 Bytes in Binary)` - Choose your signature file.
 
+### PS Passthrough
+
+![GP2040-CE Configurator - PS Passthrough](assets/images/gpc-add-ons-ps-passthrough.png)
+
+Enabeling this add-on will allow you to use a licenced 3rd party device to authenticate off of.  This addon requires that you have something like the ![USB Passthrough Board](https://github.com/OpenStickCommunity/Hardware/tree/main/USB%20Passthrough%20Board) or a board with a USB passthrough port on it already.  If you have passthrough enabled you can turn off the above `PS4 Mode` addon as the two will not work together.  Please also ensure that under the `Settings` section you have chosen PS4 mode and picked if you want the GP2040-CE unit to function as a controller or as a fightstick.   
+
+* `D+` - Choose the RP2040 GPIO to be used for Data +.
```

Comment:
Because of the similarities between the Keyboard Host Passthrough and the PS Passthrough, do we want to have unified language between the two? Perhaps even have them next to one another so that the Example Wiring Diagram can be shared between the two.